buildenv/jenkins/JenkinsfileBase

```diff
@@ -1304,4 +1293,17 @@ def getGitRepoBranch(ownerBranch, defaultOwnerBranch, repo) {
 	return actualRepoBranch
 }
 
+def forceCleanWS() {
+	try {
+		cleanWs disableDeferredWipeout: true, deleteDirs: true
```

Comment:
```suggestion
	cleanWs disableDeferredWipeout: true, deleteDirs: true
```

Reply:
I have no idea about this space issue. Everything looks fine in my local editor. However it seems there are spaces doubled in github repo. Also look at the current JenkinsfileBase  file all the other code has the same issue.